src/components/login/loginForm/LoginForm.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Formik } from 'formik';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import './loginForm.scss';
+import { useDispatch, useSelector } from 'react-redux';
+import { setAuthUserData } from '../../../store/reducers/login-reducer';
+import * as Yup from 'yup';
+
+//filtering and checking what the user has entered into forms
+const validationSchema = Yup.object().shape({
+    email: Yup.string()
+        .email('Invalid email')
+        .required('Required'),
+    password: Yup.string()
+        .required('Required')
+});
+
+const LoginForm = () => {
+
+    const stateEmail = useSelector((state) => state.login.email);
+    const statePassword = useSelector((state) => state.login.password);
+    const stateIsAuth = useSelector((state) => state.login.isAuth);
+    console.log(stateEmail);
+    console.log(statePassword);
+    console.log(stateIsAuth);
+
+    const dispatch = useDispatch();
+
+    //styles from material ui
+    const useStyles = makeStyles((theme) => ({
+        root: {
+            '& > *': {
+                margin: theme.spacing(1),
+                width: '25ch',
+            },
+        },
+    }));
+    const classes = useStyles();
+
+    // function 'submit' where will be all actions after submitting
+    const submit = (values, { setSubmitting }) => {
+        alert(values.email, values.password);
+        dispatch(setAuthUserData(values.email, values.password));
+        setSubmitting(false);
+    };
+
+    return <div className='loginForm'>
+        <Formik initialValues={{ email: '', password: '' }} validationSchema={validationSchema}
+            onSubmit={submit} >
+
+            {({ values, errors, touched, handleChange, handleBlur, handleSubmit, isSubmitting, }) => (
+
+                <form onSubmit={handleSubmit} className={classes.root} noValidate autoComplete="off">
+                    {/* form for email */}
+                    <div>
+                        <TextField id="outlined-basic" label="Mail" variant="outlined" type="email"
+                            name="email" onChange={handleChange} onBlur={handleBlur} value={values.email} />
+                    </div>
+                    {errors.email && touched.email && errors.email}
```

Comment:
use <ErrorMessage > from formik instead of {errors.email && touched.email && errors.email}

![image](https://user-images.githubusercontent.com/86973575/125136851-43eaf800-e114-11eb-9422-80b733f499f6.png)